views/js/controller/DeliveryServer/awaiting.js

```diff
@@ -47,6 +47,44 @@ define([
      */
     var cssScope = '.awaiting-authorization';
 
+    /**
+     * Nothing needs to be taken to system clipboard on proceed action
+     */
+    var cleanClipboard = function cleanClipboard () {
+        var textToClipboard = ' ';
+        var selected;
+        var success;
+        var el;
+        if (window.clipboardData) { // Internet Explorer
+            window.clipboardData.setData ("Text", textToClipboard);
+        } else {
+            el = document.createElement('textarea');  // Create a <textarea> element
+            loadingBar.stop();
```

Comment:
Why is it required to stop the loadingBar?

Reply:
yep, forgot to delete, I've moved this content from the test runner, good catch =)